pages/ksphere/konvoy/1.5/release-notes/software-sources/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: layout.pug
+title: Software Sources
+menuWeight: 100
+excerpt: List of source software sources under GPL, and MPL used by D2IQ Konvoy
+render: mustache
+---
+
+The table below lists the Sources for the Free Software under GNU General Public License, and Mozilla Public License
```

Comment:
Is this different from https://docs.d2iq.com/ksphere/konvoy/1.4/release-notes/open-source-attribution/?

Reply:
Yeah,  different.

> "lists the Sources"

Reply:
Only different in that these items need additional information.